W-16026636 Making fixes (#341)
mulesoft/docs-access-management: modules/ROOT/pages/conf-openid-connect-task.adoc

= Configuring OpenID Connect Identity Management
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> There are two identity management procedures for client registration in OpenID Connect:

Reviewer: ```suggestion
You can use one of these identity management procedures for client registration in OpenID Connect:
```

* Dynamic registration
* Manual registration

Anypoint Platform supports the following registrations:

* Clients created dynamically in Okta and OpenAM
* Clients created manually in Salesforce, Okta, OpenAM, and PingFederate

> Although integration with the aforementioned Identity providers has been officially tested, Anypoint Platform supports the OpenID Connect Protocol. This means that any Identity Provider that supports the protocol can integrate unless they diverge from the http://openid.net/specs/openid-connect-core-1_0.html[specification].

Reviewer: ```suggestion
MuleSoft has officially tested the integration with these identity providers. Because Anypoint Platform supports the OpenID Connect Protocol, any identity provider that supports the protocol can integrate unless they diverge from the http://openid.net/specs/openid-connect-core-1_0.html[specification].
```
